doc/source/merging.rst

```diff
@@ -1132,4 +1104,121 @@ values inplace:
    @savefig merging_update.png
    p.plot([df1_copy, df2], df1,
           labels=['df1', 'df2'], vertical=False);
-   plt.close('all');
\ No newline at end of file
+   plt.close('all');
+
+.. _merging.time_series:
+
+Timeseries friendly merging
+---------------------------
+
+.. _merging.ordered_merge:
+
+Merging Ordered Data
+~~~~~~~~~~~~~~~~~~~~
+
+The ``ordered_merge`` function allows combining time series and other
+ordered data. In particular it has an optional ``fill_method`` keyword to
+fill/interpolate missing data:
+
+.. ipython:: python
+
+   left = DataFrame({'k': ['K0', 'K1', 'K1', 'K2'],
+                     'lv': [1, 2, 3, 4],
+                     's': ['a', 'b', 'c', 'd']})
+
+   right = DataFrame({'k': ['K1', 'K2', 'K4'],
+                      'rv': [1, 2, 3]})
+
+   result = ordered_merge(left, right, fill_method='ffill', left_by='s')
+
+.. ipython:: python
+   :suppress:
+
+   @savefig merging_ordered_merge.png
+   p.plot([left, right], result,
+          labels=['left', 'right'], vertical=True);
+   plt.close('all');
+
+.. _merging.asof_merge:
+
+Merging AsOf
+~~~~~~~~~~~~
+
+.. versionadded:: 0.18.2
+
+Similar to an ``pd.ordered_merge()``, an ``pd.asof_merge()`` allows one to propogate forward
+values that are missing. ``pd.asof_merge()`` allows some specific options on this propogation
```

Comment:
This propagates forward values that are *not* missing, right? Also, not sure what the second sentence is saying.